doc/source/whatsnew/v0.18.2.txt

```diff
@@ -74,3 +74,8 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
+
+- Bug in ``.resample(..)`` with a ``PeriodIndex`` not changing its ``freq``
+appropriately when empty (:issue:`13067`)
+- Bug in ``PeriodIndex`` construction returning a ``float64`` index in some
```

Comment:
OK, tbc this had no real user facing impact - it still looked and behaved as a PeriodIndex, but underlying was float rather than int. Will add a test; let me know if you still think we need an example in the docs